README.md

```diff
@@ -3,4 +3,42 @@
 This will be an official client for [Apify API](https://www.apify.com/docs/api/v2).
 It's still work in progress, so please don't use it yet!
 
+TODO: write a proper README
 
+## Installation
+
+Requires Python 3.7+
```

Comment:
out of interest, is the client expected to work with 3.8 as well?

Reply:
Yup, it should be forward-compatible, it would be pretty weird if it wasn't.